Tribler/Test/Core/APIImplementation/test_maketorrent.py

```diff
@@ -0,0 +1,19 @@
+# coding=utf-8
+import os
+
+from Tribler.Core.APIImplementation.maketorrent import pathlist2filename
+from Tribler.Test.test_as_server import BaseTestCase
+
+
+class TestMakeTorrent(BaseTestCase):
```

Comment:
Maybe it's better to derive from `AbstractServer` (so we have the reactor checks and a temporary directory)?